structuralcodes/sections/specific.py

```diff
@@ -0,0 +1,446 @@
+
+    def bending_strength_yp(self, N: float = 0) -> float:
+        """Returns the beding strength in y+ direction for a given
+        value of axial force (+: tension, -: compression).
+        """
+        return 0.0
+
+    def bending_strength_yn(self, N: float = 0) -> float:
+        """Returns the beding strength in y- direction for a given
+        value of axial force (+: tension, -: compression).
+        """
+        return 0.0
+
+    def _arrange_reinforcement_in_section(self):
+        """The method creates the arrays for reinforcements
+        starting from reinforcement_data.
+        """
+        delta = 2 * (self.cover + self.stirrups_data.diameter / 2.0)
+        hc = self.height - delta
+        wc = self.width - delta
+        delta = (
+            self.reinforcement_data.diameter_corner
+            + self.stirrups_data.diameter
+        )
+        hcc = hc - delta
+        wcc = wc - delta
+        xmin = -wcc / 2.0
+        xmax = wcc / 2.0
+        ymin = -hcc / 2.0
+        ymax = hcc / 2.0
+        self.xs = [xmin, xmax, xmax, xmin]
+        self.ys = [ymin, ymin, ymax, ymax]
+        self.d = [self.reinforcement_data.diameter_corner] * 4
+        # bottom bars
+        for i in range(self.reinforcement_data.number_bottom):
+            self.xs.append(
+                xmin
+                + (xmax - xmin)
+                / (self.reinforcement_data.number_bottom + 1)
+                * (i + 1)
+            )
+            self.ys.append(ymin)
+            self.d.append(self.reinforcement_data.diameter_bottom)
+        # top bars
+        for i in range(self.reinforcement_data.number_top):
+            self.xs.append(
+                xmin
+                + (xmax - xmin)
+                / (self.reinforcement_data.number_top + 1)
+                * (i + 1)
+            )
+            self.ys.append(ymax)
+            self.d.append(self.reinforcement_data.diameter_top)
+        # left bars
+        for i in range(self.reinforcement_data.number_left):
+            self.xs.append(xmin)
+            self.ys.append(
+                ymin
+                + (ymax - ymin)
+                / (self.reinforcement_data.number_left + 1)
+                * (i + 1)
+            )
+            self.d.append(self.reinforcement_data.diameter_left)
+        # right bars
+        for i in range(self.reinforcement_data.number_right):
+            self.xs.append(xmax)
+            self.ys.append(
+                ymin
+                + (ymax - ymin)
+                / (self.reinforcement_data.number_right + 1)
+                * (i + 1)
+            )
+            self.d.append(self.reinforcement_data.diameter_right)
+
+        self.xs = np.array(self.xs)
+        self.ys = np.array(self.ys)
+        self.d = np.array(self.d)
+        # compute Areas for each reinforcement
+        self.As = np.pi * (self.d**2) / 4
+
+        self.xmin = xmin
+        self.xmax = xmax
+        self.ymin = ymin
+        self.ymax = ymax
```

Comment:
I would remove these lines for now (see general comment)

Reply:
Removed